lomond/opcode.py

```diff
@@ -41,21 +41,10 @@ def to_str(cls, opcode):
             }
         return cls._opcode_to_str.get(opcode, '?')
 
-
 reserved_opcodes = {
-    Opcode.RESERVED1,
-    Opcode.RESERVED2,
-    Opcode.RESERVED3,
-    Opcode.RESERVED4,
-    Opcode.RESERVED5,
-    Opcode.RESERVED6,
-    Opcode.RESERVED7,
-    Opcode.RESERVED8,
-    Opcode.RESERVED9,
-    Opcode.RESERVED10,
+    Opcode.__dict__[x] for x in Opcode.__dict__.keys() if x.startswith('RESERVED')
```

Comment:
The original is a bit verbose, I'll grant you. But I don't mind the repetition here. The duplication is in a single file, and a literal has the advantage that it doesn't execute any code at import time. It's also unlikely to change any time soon, as far as I know the websocket protocol has stabilised.

Reply:
> 
> 
> The original is a bit verbose, I'll grant you. But I don't mind the repetition here. The duplication is in a single file, and a literal has the advantage that it doesn't execute any code at import time. It's also unlikely to change any time soon, as far as I know the websocket protocol has stabilised.

Yes, I noticed that the Websockets has not changed since 2011.. 

I enjoyed reading the code and just could not help but make the change :).. habit